scripts/check-test-determinism.py

```
@@ -316,6 +356,237 @@ def _is_assertion_line(line: str) -> bool:
+            quote_delimiter = _quote_delimiter_at(line, index)
+            for delimiter_index in range(index, min(index + len(quote_delimiter), len(line))):
+                executable[delimiter_index] = False
+            string_kind = (
+                "f-string-text"
+                if _python_f_string_starts_at(line, index)
+                else "string-text"
+            )
+            contexts.append((string_kind, 0, quote_delimiter))
+            index += len(quote_delimiter)
+            continue
+        elif character == "`":
```

Ruff (0.16.1)

[warning] Unnecessary `elif` after `continue` statement

Remove unnecessary `elif`

(RET507)